src/main/java/com/github/juliarn/npc/modifier/TeleportModifier.java

```diff
@@ -0,0 +1,76 @@
+package com.github.juliarn.npc.modifier;
+
+import com.comphenix.protocol.PacketType.Play.Server;
+import com.comphenix.protocol.events.PacketContainer;
+import com.github.juliarn.npc.NPC;
+import org.bukkit.Location;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * A teleport modifier for a player
+ *
+ * @since 2.7-SNAPSHOT
+ * @author unldenis <https://github.com/unldenis>
```

Comment:
Unsure about the author tag, thats a thing @juliarn must decide.

Reply:
I don't have anything against it. but seems like the javadoc can't compile because of something is wrong with it.